lib/ui/window/platform_configuration.cc

```diff
@@ -35,7 +32,15 @@ PlatformConfigurationClient::~PlatformConfigurationClient() {}
 
 PlatformConfiguration::PlatformConfiguration(
     PlatformConfigurationClient* client)
-    : client_(client) {}
+    : client_(client) {
+  if (client_->ImplicitViewEnabled()) {
+    // Add PlatformConfiguration's window now, but don't add dart:ui's View
+    // here. The dart:ui needs another way to add the implicit view
+    // synchronously so that the view is available before the main function.
+    // See _implicitViewEnabled in natives.dart.
+    AddWindowRecord(kFlutterImplicitViewId);
```

Comment:
`AddWindowRecord` uses `library_` which isn't set until `DidCreateIsolate`. 

Can we move this logic to `DidCreateIsolate`?

Reply:
That's right! I wonder why it didn't cause any errors.

Reply:
Oh, because I pass in the reference to `library_` and the value that it contains is changed. I don't think the movement is necessary, and putting it at the constructor might be clearer.

Reply:
Oh but if we put it here we don't need a reference any more...

Reply:
I'm keeping my implementation since it's safe enough.